lib/elephant_in_the_room_web/controllers/image_controller.ex

```diff
@@ -20,13 +36,19 @@ defmodule ElephantInTheRoomWeb.ImageController do
     image = HTTPoison.get!(url, [], hackney: [{:follow_redirect, true}])
 
     type =
-      :proplists.get_value("Content-Type", image.headers)
+      :proplists.get_value("content-type", conn.req_headers)
       |> String.split("/")
       |> List.last()
 
-    {:ok, saved_image} = Sites.create_image(%{"binary" => image.body, "type" => type})
+    name = generate_name(type)
```

Comment:
shouldn't we check first 'content-type' to be an image? i.e. jpg or .png. I was able to upload any type file.